turbine_tank/run_tank.py

```diff
@@ -0,0 +1,63 @@
+# Copyright 2024 Advanced Micro Devices, Inc
+#
+# Licensed under the Apache License v2.0 with LLVM Exceptions.
+# See https://llvm.org/LICENSE.txt for license information.
+# SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception
+
+import argparse
+import unittest
+import tank_util
+
+import turbine_models.tests.sd_test as sd_test
+import os
+from turbine_models.turbine_tank import turbine_tank
+
+import pytest
+
+parser = argparse.ArgumentParser()
+parser.add_argument(
+    "--download_ir",
+    action=argparse.BooleanOptionalAction,
+    default=False,
+    help="download IR from turbine tank",
+)
```

Comment:
Basically, this is to download the latest mlir files for all the models from Azure to a local cache. I can make it a separate file then (just wanted to centralize all turbine tank capabilities).

Reply:
I'll keep it centralized for now because downloading is another turbine tank feature, so having it in the run_tank.py would be nice to be one centralized file where you can call all features. The turbine_tank file in SHARK-Turbine has both upload and download features baked into it, so we are just calling those here.